README.md

```diff
@@ -2,162 +2,102 @@
 
 Time series box, line plots and heatmaps for every available season
 
-## Installation
 
-### Easy deploy with Docker
+### Set up a local development environment
```

Comment:
This could go under a section `## Development`

Then explain at a high level what steps will be required. (R, R libraries, other dependencies)